lib/wadler/print.rb

```diff
@@ -59,6 +64,10 @@ def nest(indent = 2, break_type = Oppen::Token::BreakType::CONSISTENT, &block)
           Oppen.begin_inconsistent(offset: indent)
         end
 
+      if block.nil?
+        raise LocalJumpError
```

Comment:
It's better to raise an argument error:

```ruby
raise ArgumentError, "Block expected but none given" if !block_given?
```

And test that we raise it when no block is given.